packages/badge/src/StatusBadge.stories.tsx

```diff
@@ -54,3 +64,73 @@ export const LanguageExamples = () => {
 		</Flex>
 	);
 };
+
+export const InTable: ComponentStory<typeof Table> = (args) => {
+	const data = [
+		{
+			id: 'RE4321–2201–03',
+			businessName: 'Orange Meat Works',
+			type: 'Record keeping—Minor',
+			status: 'Pending',
+		},
+		{
+			id: 'RE4321–2201–02',
+			businessName: 'Orange Meat Works',
+			type: 'Hygiene—Major',
+			status: 'Open',
+		},
+		{
+			id: 'RE4321–2201–01',
+			businessName: 'Molong Meat Works',
+			type: 'Record keeping—Minor',
+			status: 'Open',
+		},
+		{
+			id: 'RE4321–2201–00',
+			businessName: 'Orange Meat Works',
+			type: 'Record keeping—Minor',
+			status: 'Closed',
+		},
+	] as const;
+
+	const toneMapper = {
+		Closed: 'success',
+		Open: 'warning',
+		Pending: 'info',
+		Draft: 'neutral',
+	} as const;
+
+	return (
+		<TableWrapper>
+			<Table {...args}>
```

Comment:
Can we remove the configurable of this story please

```suggestion
			<Table>
```